diff -ur openPMD-api-0.16.0.orig/CMakeLists.txt openPMD-api-0.16.0/CMakeLists.txt
--- openPMD-api-0.16.0.orig/CMakeLists.txt	2024-10-07 20:36:35.000000000 -0500
+++ openPMD-api-0.16.0/CMakeLists.txt	2024-11-02 11:29:25.684277242 -0500
@@ -314,7 +314,7 @@
 string(CONCAT openPMD_HDF5_STATUS "")
 # version: lower limit
 if(openPMD_HAVE_HDF5)
-    if(HDF5_VERSION STREQUAL "")
+    if("${HDF5_VERSION}" STREQUAL "")
         message(WARNING "HDF5_VERSION is empty. Now assuming it is 1.8.13 or newer.")
     else()
         if(HDF5_VERSION VERSION_LESS 1.8.13)
